frontend/src/components/filters/FiltersWarningTS.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { QuestionCircleOutlined } from "@ant-design/icons";
+import { Tooltip, Typography } from "antd";
+import FiltersInfo from "./FiltersInfoTS";
+const { Text } = Typography
+
+interface FiltersWarning {
+    filters: any,
```

Comment:
There is a type defined for filters, so you should use that, not any. See `FilterSet` in `paged_items.ts`.